### ISSUE TITLE
Unable to install React Native Pendo in Apple silicon for IOS

### ISSUE THREAD

**Issue by vikas-0** [pendo-io/pendo-mobile-ios#4]
after pod install, When I run `react-native run-ios`, I get following error 

```
Create build description
Build description signature: <signature>
Build description path: /Users/<username>/Library/Developer/Xcode/DerivedData/<Project>/Build/Intermediates.noindex/XCBuildData/07aa4a489793fd54166-desc.xcbuild

note: Build preparation complete
note: Building targets in dependency order
/Users/<user>/Documents/Developement/mobile/ios/<Project>.xcodeproj: error: The linked library 'libPods-<Project>.a' is missing one or more architectures required by this target: x86_64. (in target '<Project>' from project '<Project>')
```

it is Apple Silicon Mac M1,

**Comment by vikas-0** [comment]
Also, the repo mentioned in https://www.npmjs.com/package/rn-pendo-sdk/v/2.13.1 is inaccessible. is it supposed to be like that?

**Comment by MikePendo** [comment]
yes, we are _not_ open source.

**Comment by MikePendo** [comment]
@vikas-0 
Have you been able to try one of the proposed work arounds above?

**Comment by vikas-0** [comment]
> @vikas-0 Have you been able to try one of the proposed workarounds above?

No, I couldn't find the time to try. We decided to go ahead for now without pendo. Perhaps, I'll look into it again, next week.

**Comment by vikas-0** [comment]
I tried the workaround, I am still getting the same error.

**Comment by MikePendo** [comment]
@vikas-0 
So I have looked again at the issue, As I mentioned we currently ship our native library as _fat static library_ which will not work for the M1 as it can't contain two slices for arm64 (simulator, device)and that what cause the issue.
The proper solution will be to ship it under XCframeworks,  I will create a pr tmrw to fix this issue. (already tried it on my M1 and it looks fine), so I believe next week we will be able to release it.
For right now the only solution I tried and it worked for me was to use Rosseta.
I will update as soon as we release.
Thanks
